<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kbvRenameClass</class>
 <widget class="QDialog" name="kbvRenameClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>349</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="renButtonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="renGroupPreview">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>551</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="title">
    <string notr="true">Preview</string>
   </property>
   <widget class="QLabel" name="renPreview1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>531</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true">TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="renPreview2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>531</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true">TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="renGroupName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="title">
    <string notr="true">Name</string>
   </property>
   <widget class="QLabel" name="renPrefixLbl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true">Prefix &amp;Name Count or Prefix Count Name</string>
    </property>
    <property name="buddy">
     <cstring>renNameEdit</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="renNameEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>256</number>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="renSuffixLbl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true">&amp;Postfix</string>
    </property>
    <property name="buddy">
     <cstring>renSuffixEdit</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="renSuffixEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>256</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="renGroupCount">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="title">
    <string notr="true">Count</string>
   </property>
   <widget class="QLabel" name="renStartLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&amp;Start value</string>
    </property>
    <property name="buddy">
     <cstring>renStartSpin</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="renStartSpin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QLabel" name="renStepLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>S&amp;tep value</string>
    </property>
    <property name="buddy">
     <cstring>renStepSpin</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="renStepSpin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="renKeepExtCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true">Keep file extension</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="renKeepExtEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>renNameEdit</tabstop>
  <tabstop>renSuffixEdit</tabstop>
  <tabstop>renStartSpin</tabstop>
  <tabstop>renStepSpin</tabstop>
  <tabstop>renKeepExtCheckBox</tabstop>
  <tabstop>renKeepExtEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
